package.json

```
@@ -20,7 +20,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint \"**/*.js\" --fix"
+    "lint": "eslint src --fix"
```

```suggestion
    "lint": "eslint src test --fix"
```
Do code suggestions still work?

---

They do, but this will break that command.

(Also, something I just though of - shouldn't we have a `lint` command that didn't `--fix` for CI?)

---

There is a `test` folder inside the `src` folder, you can move that folder one level above.

Also regarding CI, I thought that already existed, but it seems not, you can add it

---

I know I could, but I didn't know if I should, hence my previous questions 😅 
Going to do that now, then.

Yup, sounds good. Will add it and rename this one to `lint-fix` most likely

---

Don't forget to make the -fix one the default when pre-committing to make our lives easier!

---

> Also regarding CI, I thought that already existed, but it seems not, you can add it

I think that it does not exist since there is no CI configured atm, see #25